docs/Helm.md

```diff
@@ -0,0 +1,106 @@
+# Splunk Operator Helm Installation
+
+## Splunk Operator Helm chart Repository
+
+Add the Splunk Operator and Enterprise charts to your Helm repository.
+
+```
+helm repo add splunk https://splunk.github.io/splunk-operator/
+helm repo update
+```
+
+The ```splunk``` chart repository contains the ```splunk/splunk-operator``` chart to deploy the Operator and the ```splunk/splunk-enterprise``` chart to deploy Splunk Custom Resources.
+
+Helm provides a long list of commands to manage your deployment, we'll be going over a few useful ones in the sections to come. You can learn more about supported commands [here](https://helm.sh/docs/helm/helm/).
+
+## Splunk Operator deployments
+
+To install the Splunk Operator:
+
+```
+helm install <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+To upgrade the operator, specify override configurations through the CLI or a new file:
+
+```
+helm upgrade --set splunkOperator.clusterWideAccess=false <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Here we've upgraded the operator by revoking cluster-wide access.
+
+To rollback the operator to a previous release:
+
+```
+helm rollback <RELEASE_NAME> <REVISION_NUMBER> -n <RELEASE_NAMESPACE>
+```
+
+To clean-up resources and uninstall the operator:
+
+```
+helm uninstall <RELEASE_NAME> -n <RELEASE_NAMESPACE>
+```
+
+### Configuring Splunk Operator deployments
+
+There are a couple ways you can configure your operator deployment
+
+1. Using a ```new_values.yaml``` file to override default values (Recommended)
+```
+helm install -f new_values.yaml <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+2. Using the Helm CLI to set new values
+```
+helm install --set <KEY>=<VALUE> <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Read more about configuring values [here](https://helm.sh/docs/intro/using_helm/).
+
+## Splunk Enterprise deployments
+
+The Splunk Enterprise chart allows you to install and configure Splunk Enterprise custom resources.
+
+First build the Splunk Operator chart as a dependency:
+```
+helm dependency build splunk/splunk-enterprise
```

Comment:
Do we need this per this statement - `This chart installs the Splunk Operator as a dependency by default.` below?

Reply:
Dependency build packages the ```splunk/splunk-operator``` chart within a sub-directory of the ```splunk/splunk-enterprise``` chart. It does not install the Splunk Operator. If the ```splunk/splunk-operator``` dependency is enabled and the dependencies have not been built, then an error will occur.

Reply:
In that case can we edit the statement above to reflect the dependency i.e `The Splunk Enterprise chart allows you to install and configure Splunk Enterprise custom resources. The splunk/splunk-enterprise chart has the splunk/splunk-operator chart as a dependency by default. To satisfy the dependencies please use the following command: ` ? 

With this the statement `This chart installs the Splunk Operator as a dependency by default.` would no longer be needed